certified-connectors/PowerBI Robots/README.md

```diff
@@ -0,0 +1,14 @@
+## PowerBI Robots - API Connector
+PowerBI Robots connector to execute the internal API across the multiple Power Platform products
```

Comment:
@AntonioArantesDevscope We need to review the connector's title internally, We will get back to you in 1-2 days.

Reply:
PowerBI Robots is the name of DevScope S.A. product, if it helps you anyway.

Reply:
@AntonioArantesDevscope For non-Microsoft publishers, It is not allowed to have Microsoft and Power platform product names in the connector titles. I would suggest you to have your connector title something like "Devscope Robots for PowerBI" or similar to it.

Docs: [Give your connector a title](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)

Reply:
@sriyen-msft We've changed to "DevScope PowerBI Robots", is that ok? Anyway, your suggestion "DevScope Robots for PowerBI" also contains the "PowerBI" keyword... If our new name "DevScope PowerBI Robots" fails we will accept your suggestion as name for our connector. Many thanks

Reply:
@AntonioArantesDevscope Thank you so much for considering my suggestion, It is still not allowed to have any Power platform product name in the title due to legal issues. We are waiting on Microsoft legal team's approval for this.